src/molecules/FeatureSquare/index.js

```diff
@@ -28,10 +29,12 @@ function FeatureSquare(props) {
 
   return (
     <div className={classnames(...classes)}>
-      <Icon icon={icon} className={styles['icon']} />
-      <Text type={4} semibold>{ header }</Text>
-      <Text type={6} className={styles['subheader']}>{ subheader }</Text>
-      <Button onClick={onClick}>{ buttonText }</Button>
+      <Icon icon={icon} className={styles['icon']} width='100' />
```

Comment:
No example for this component?

Reply:
Nahh, moving fast -- can go back and add it.